### DISCUSSION TITLE
Is it possible to configure shell allowlist to allow any shell command execution?

### DISCUSSION THREAD

**Discussion by fogine** [tauri-apps/tauri#4557] [Q&A]
We are developing internal tool for developers where the user is able to configure any arbitrary command to be executed in the UI.
I've following in `tauri.conf.json`

```json
"allowlist": {
          "shell": {
              "execute": true,
              "all": true,
              "open": "^.*$"
      }

}

```
But I'm still getting `Program not allowed on the configured shell scope` error.

**Comment by FabianLars** [comment]
For others looking at this: #4575
